CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ rewritten to their query representation
 - Add cli option -e to load PartiQL literal data as default catalog data
 
 ### Changed
+- Migrate the usage of `PartiQLValue` to `Datum` in conformance tests
```

Comment:
Probably can leave out this entry from the changelog since it doesn't affect the public API or any behaviors of the public APIs